ynab_sdk/utils/configurations/cached.py

```diff
@@ -3,8 +3,9 @@
 
 class CachedConfig(DefaultConfig):
 
-    def __init__(self, redis_host: str, redis_port: int, redis_db: int = 0, **kwargs):
+    def __init__(self, redis_host: str, redis_port: int, redis_db: int = 0, redis_pass: str = None, **kwargs) -> None:
```

Comment:
`redis_pass` should be typed as Optional[str]

Reply:
Done